esp-hal/src/parl_io.rs

```diff
@@ -1579,6 +1603,51 @@ where
     }
 }
 
+/// Creates a TX channel
+#[non_exhaustive]
```

Comment:
I don't think these need to be non_exhaustive because they don't have public fields?

Reply:
true - I think I made them first non-exhaustive and then made the fields private or something like that 👍